### PR TITLE
Use Neo4j driver `to_df()` method

### PR DESCRIPTION
**Pull request by adamnsch**

### CODE REVIEW

**Comment by adamnsch** [comment]
It's failing due to some unmet dependencies. I will update the docker image

**Comment by adamnsch** [comment]
> It's failing due to some unmet dependencies. I will update the docker image

Done

File: graphdatascience/tests/integration/test_model_object.py [hunk 1]
```diff
@@ -54,3 +56,39 @@ def test_model_drop(gds: GraphDataScience, G: Graph) -> None:
...
+def test_model_loaded(gs_model: GraphSageModel) -> None:
+    assert gs_model.loaded()
+
+
+def test_model_stored(gs_model: GraphSageModel) -> None:
+    assert not gs_model.stored()
+
+
+def test_model_creation_time(gs_model: GraphSageModel) -> None:
+    assert gs_model.creation_time().year > 2000
```
Comment (knutwalker) [inline graphdatascience/tests/integration/test_model_object.py:86]: not so Back To The Future compatible 😅

**Review by knutwalker — APPROVED** [review]
🐼

**Comment by adamnsch** [comment]
> panda_face

Thanks for review! :heart: Agree with your remark on the docs - will fix

**Pull request merged**